ruby/example_code/dynamodb/README.md

```diff
@@ -8,11 +7,11 @@ Shows how to use the AWS SDK for Ruby to work with Amazon DynamoDB.
 <!--custom.overview.start-->
 <!--custom.overview.end-->
 
-*DynamoDB is a fully managed NoSQL database service that provides fast and predictable performance with seamless scalability.*
+_DynamoDB is a fully managed NoSQL database service that provides fast and predictable performance with seamless scalability._
 
 ## ⚠ Important
 
-* Running this code might result in charges to your AWS account.
+* Running this code might result in charges to your AWS account. For more details, see [AWS Pricing](https://aws.amazon.com/pricing/?aws-products-pricing.sort-by=item.additionalFields.productNameLowercase&aws-products-pricing.sort-order=asc&awsf.Free%20Tier%20Type=*all&awsf.tech-category=*all) and [Free Tier](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc&awsf.Free%20Tier%20Types=*all&awsf.Free%20Tier%20Categories=*all).
```

Comment:
Remove query strings which may add value but are prone to break. The base URL's work fine.
```suggestion
* Running this code might result in charges to your AWS account. For more details, see [AWS Pricing](https://aws.amazon.com/pricing/?aws-products-pricing) and [Free Tier](https://aws.amazon.com/free/?all-free-tier).
```